source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -83,13 +86,55 @@ the SRV record. For example, when using the non-FQDN ``servername`` and the
 default domain set to ``example.com``, a Client should end up requesting the
 SRV record for ``servername.example.com``.
 
+Default Connection String Options
+---------------------------------
+
+A Client MUST also query the DNS server for TXT records. If available, these
+TXT records provide default connection string options. In most cases only one
+TXT record is necessary, unless the default options exceed the maximum length
+of 255 characters that TXT records allow for. To allow for this case, a Client
+MUST allow for multiple TXT records for the same host name and consider the
+options in all of them.
```

Comment:
Optionally, we could support "chunking" extra long options by supporting multiple options with the same name. e.g.

```
someArray = [1, 2, 3, 4, 5, 6, 6, 7 ...
someArray = 10000, 10001, 10002...]
```

could be parsed as a single variable.  What do you all think?

Reply:
You can't guarantee the order in which TXT records come back in, so that means drivers need to do some tricky logic to put them together in the right way.

I also don't think this is needed.

Reply:
Agreed. Let's not do anything to support a single key-value pair that exceeds 255 chars.